settings.py

```diff
@@ -1,4 +1,16 @@
-site_url = "http://squ-test.nodeconductor.com"
-username = 'Julia'
-password = '123'
-user_full_name = 'Trygubniak'
\ No newline at end of file
+# site_url = "http://squ-test.nodeconductor.com"
+# username = 'Julia'
+# password = '123'
+# user_full_name = 'Trygubniak'
+
+site_url = "http://web-test.nodeconductor.com"
+username = 'Alice'
+password = 'Alice'
+user_full_name = 'Alice Lebowski'
+nec_organization = 'Ministry of Bells'
+project_name = 'OpenStack test project'
+key_name = 'Openstack test key'
+resource_name = 'OpenStack test instance'
+time_wait = 180
+time_wait_again = 60
```

Comment:
settings parameters `time_wait` and `time_wait_again` look strange for me: it is hard to guess what do they mean from their names.